Stack/Opc.Ua.Core/Types/Encoders/JsonEncoder.cs

```diff
@@ -498,38 +501,45 @@ public void PopNamespace()
             m_namespaces.Pop();
         }
 
-        private static readonly char[] m_specialChars = new char[] { s_quotation, s_backslash, '\n', '\r', '\t', '\b', '\f', };
-        private static readonly char[] m_substitution = new char[] { s_quotation, s_backslash, 'n', 'r', 't', 'b', 'f' };
+        private static readonly Dictionary<char, string> m_substitution = new Dictionary<char, string>
+        {
+            { s_quotation, "\\\"" },
+            { s_backslash, "\\\\" },
+            { '\n', "\\n" },
+            { '\r', "\\r" },
+            { '\t', "\\t" },
+            { '\b', "\\b" },
+            { '\f', "\\f" }
+        };
 
+        /// <summary>
+        /// Escapes a string and writes it to the stream.
+        /// </summary>
+        /// <param name="value"></param>
         private void EscapeString(string value)
```

Comment:
as discussed, if you can reproduce the new benchmark results, create one func for platforms which do not support `StreamWriter.Write(ReadOnlySpan<byte>)` and one optimized for Span.

Reply:
added

Reply:
Why did you not use EscapeStringSpanChars or EscapeStringSpanCharsInline? Was this faster?

Reply:
I am using EscapeStringSpanCharsInline() ,the name in the JSonEncoder is EscapeString(). if the conditional for .net is true it uses the inline logic else uses the legacy_plus logic

Reply:
updated to inline version.